validator/tests/mock.rs

```diff
@@ -174,10 +174,17 @@ async fn four_validators_can_do_skip_block() {
         spawn(validator);
     }
 
-    // Wait for the new block producer to create a block.
+    // Wait for the new block producer to create a micro block.
```

Comment:
I don't get the change. I guess that next validator is popped when calling `pop_validator_for_slot` and such validator is getting paused when calling `tokio::time::pause` so the rest should produce a skip block right after. Or am I missing something?